micrometer-core/src/main/java/io/micrometer/core/instrument/config/MeterFilter.java

```diff
@@ -414,7 +414,7 @@ public DistributionStatisticConfig configure(Meter.Id id, DistributionStatisticC
 
     /**
      * Enables the provided filter for the Meters selected by the predicate.
-     * <code>registry.config().meterFilter(MeterFilter.forMeters(id -> id.getName().startsWith("test"), MeterFilter.ignoreTags("ignored")))</code>
+     * <code>registry.config().meterFilter(MeterFilter.forMeters(id -&gt; id.getName().startsWith("test"), MeterFilter.ignoreTags("ignored")))</code>
```

Comment:
I don't understand why this JavaDoc error started happening but it was blocking the build so I fixed it. It looked like the following (although it's for the other JavaDoc with the same issue):

```
> Task :micrometer-jakarta9:javadoc FAILED
/Users/tludwig/repos/micrometer/micrometer-jakarta9/src/main/java/io/micrometer/jakarta9/instrument/jms/JmsInstrumentation.java:52: error: bad use of '>'
 * consumer.setMessageListener(message -> consumeMessage(message));
                                        ^
1 error
```

Maybe it is worth spending more time to figure out what's different in the JavaDoc configuration that's causing this.